cmake/onnxruntime_opschema_lib.cmake

```diff
@@ -0,0 +1,28 @@
+# Schema library (for contrib ops and training ops)
+
+set (CONTRIB_OPS_DIR ${ONNXRUNTIME_ROOT}/core/graph/contrib_ops)
+set (TRAINING_OPS_DIR ${ORTTRAINING_ROOT}/orttraining/core/graph)
+
+file(GLOB_RECURSE contrib_ops_schema_src
+   "${CONTRIB_OPS_DIR}/*.cc"
+   "${TRAINING_OPS_DIR}/training_op_defs.cc"
+)
+
+add_library(ort_opschema_lib ${contrib_ops_schema_src})
```

Comment:
A static library? How will it be used?

Reply:
Usually it's hard to distribute a static library binary. For example, GCC has two  So would you please tell us more about your plan?

Reply:
Talked with @gramalingam offline. We will try this approach first. The users of this library need to make sure they will use the same onnx version/protobuf version as onnx runtime.